File: Chapter2/2.3/problem1.py
from regex import R


n = int(input("Enter Your N: "))
temp = n
sum = 0

while(temp !=0):
    r = temp%10
    sum = sum*10+r
    temp = temp//10

if n==sum:
    print("Number is Pallindrome")
else:
     print("Number is No Pallindrome")

File: Chapter2/2.3/problem1.c
#include<stdio.h>

int main(){
    int num, sum=0, temp, r;
    printf("Enter N : ");
    scanf("%d", &num);

    temp = num;

    while (temp!=0)
    {
        r = temp%10;
        sum =  sum*10+r;
        temp = temp/10;
    }
    if (num == sum)
    {
        printf("Number Pallindrome");
    }
    else
        printf("Number Not Pallindrome");
    
}